flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/catalog/hive/HiveCatalogDataTypeTest.java

```diff
@@ -122,20 +121,18 @@ public void testDataTypes() throws Exception {
 	}
 
 	@Test
-	public void testNonExactlyMatchedDataTypes() throws Exception {
+	public void testNonSupportedBinaryDataTypes() throws Exception {
```

Comment:
we should break this down to two individual tests, one for BINARY and one for VARBINARY